src/coderunner/CodeRunner.ts

```diff
@@ -1,20 +1,68 @@
-import { useCallback, useEffect, useMemo, useRef, useState } from "react";
-import { TestCases, TestResults } from "../models/Tests";
 import {
```

Comment:
Standalone implementation of a class that can run Python code (without any knowledge of challenges, books etc.)